src/components/inline/DonationProjectCard/DonationProjectCard.stories.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import DonationProjectCard from "./DonationProjectCard";
+import background from "./blackbackground.jpg";
+
+export default {
+  title: "Inline/DonationProjectCard",
+  component: DonationProjectCard,
+};
+
+const Template = () => (
+  <div style={{ backgroundImage: `url(${background})` }}>
+    <DonationProjectCard title="Star Hotel SRO" subtitle="280 People" />
```

Comment:
title={args.title} subtitle={args.subtitle}

Reply:
👍 Changed to 

title={args.data.title} subtitle={args.data.subtitle}

since my default is 

DefaultDonationProjectCard.args = {
  data: {
    title: "Star Hotel SRO",
    subtitle: "280 People",
  },
};

Reply:
You can take out the data object

Reply:
👍 Deleted data object.